### DISCUSSION TITLE
[Enhancement] Consider making RefreshView available with all Layout containers

### DISCUSSION THREAD

**Discussion by PureWeen** [dotnet/maui#199] [General]
https://github.com/xamarin/Xamarin.Forms/issues/11675

## Summary
Despite it being written on the RefreshView Documentation Page, I just sunk like 3 hours in trying to make a RefreshView work with another container. I think it should not depend on scrollable content, since there's no reason to disallow a StackLayout from being refreshed, if it's not in a ScrollView.

(Also I'd suggest to remove the RefreshView and replace it with an Behavior on either layout or ContentView.)

## API Changes

Sorry - don't know.

## Intended Use Case
Allow RefreshView to be used throughout Xamarin Forms instead of only with the scrollables (especially since nesting ScrollViews is problematic)

**Comment by KPixel** [comment]
Regarding removing the RefreshView and replacing it with a Behavior, what do you think of taking it one step further and adding it to a better layout design? More details here: #138
